addOns/ascanrulesAlpha/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Update links to zaproxy repo.
 - Target 2.10 core and use new logging infrastructure (Log4j 2.x).
+- The LdapInjection scan rule was modified to make use of the Dice algorithm for calculating the match percentage, thus improving its performance.
```

Comment:
It really should be the name as show in the API/UI/help (i.e. `LDAP Injection`).